connectors/byoc.py

```diff
@@ -335,7 +318,9 @@ def __init__(self, filter_=None):
 
         self.advanced_rules = filter_.get("advanced_snippet", {})
         self.basic_rules = filter_.get("rules", [])
-        self.validation = filter_.get("validation", {"state": "", "errors": []})
+        self.validation = filter_.get(
+            "validation", {"state": FilteringValidationState.VALID.value, "errors": []}
+        )
```

Comment:
Is this change relevant to the PR? Seems like should be done separately.

Reply:
It fails one of my tests. But I can make it a separate PR.

Reply:
Made the change in a spearate PR: #495